glitch/apps.py

```diff
@@ -154,3 +173,35 @@ def glitch_video(input_path: str, output_path: str) -> None:
     reader.wait()
     writer.stdin.close()
     writer.wait()
+
+
+def swap_blocks_static(width: int, height: int, min_blocks=1, max_blocks=4,
```

Comment:
I think the name of this method is a bit misleading. It does not actually swap any block, but creates a "definition" or "instructions" to do it. Something like `create_block_swap_effect_parameters` is more clear. Also, it returns a `Dict`